project/Makefile

```diff
@@ -1,22 +1,42 @@
 # DO NOT EDIT THIS FILE!
 #
-# It's auto-generated by sonata-project/dev-kit package.
+# It's auto-generated by symfony-cmf/dev-kit package.
 
-.PHONY: test docs
```

Comment:
this should stay in: https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

Reply:
we do not have docs to test in our bundles/components.

Reply:
yeah, but unless i missed something, you completely removed `.PHONY`. this should list all meta targets that are not files (which is every target for us, as we are not compiling code with the makefile)

do a `touch test && make test` on the commandline to see the effect of having or not having phony.